packages/providers/package.json

```diff
@@ -49,10 +49,9 @@
     "@fuel-ts/utils": "workspace:*",
     "@graphql-codegen/cli": "^2.13.7",
     "@graphql-codegen/typescript": "^2.8.0",
-    "@graphql-codegen/typescript-graphql-request": "^4.5.7",
     "@graphql-codegen/typescript-operations": "^2.5.5",
-    "@types/ramda": "^0.29.3",
+    "@graphql-codegen/typescript-generic-sdk": "^3.1.0",
```

Comment:
Am I right to assume that our tests/CI would still pass even if someone deleted these `codegen-*` plugins?

I say this because there is no reference to them in any Typescript file.

I wonder if we should try validating the GraphQL generation steps.

Have you thought about it?

Reply:
Never mind, I just checked it, and the build will properly fail for missing plugins.

```
@fuel-ts/providers:prebuild: [FAILED] plugin_helpers_1.DetailedError is not a constructor
```